package.json

```diff
@@ -697,12 +697,12 @@
                 {
                     "command": "appService.Deploy",
                     "when": "explorerResourceIsFolder == true",
-                    "group": "zzz_@appServiceDeployCommands"
+                    "group": "zzz_deploy@1"
                 },
                 {
                     "command": "appService.Deploy",
                     "when": "explorerResourceIsFolder == false && resourceExtname =~ /.[j|w]ar$/",
-                    "group": "zzz_@appServiceDeployCommands"
+                    "group": "zzz_deploy@1"
```

Comment:
Can we change the name to something that makes it obvious it's cross-extension?  Otherwise someone may break them again unknowingly.

E.g. zzz_azuretools_deploy?

Reply:
Sure